src/Microsoft.Azure.WebJobs.Host/Executors/JobHostConfigurationExtensions.cs

```diff
@@ -515,6 +522,41 @@ private static Assembly GetHostAssembly(IEnumerable<MethodInfo> methods)
             }
         }
 
+        #region Hackery for ExtensionLocator
+
+        // create IConverterManager adapters to any legacy ICloudBlobStreamBinder<T>. 
+        static void AddStreamConverters(IExtensionTypeLocator extensionTypeLocator, ConverterManager cm)
```

Comment:
>AddStreamConverters [](start = 20, length = 19)

This shims the old ICloudBlobStreamBinder  onto the new converter-manager interfaces.  
Essentially demonstrating we have a fix for [https://github.com/Azure/azure-webjobs-sdk/issues/995](https://github.com/Azure/azure-webjobs-sdk/issues/995), [https://github.com/Azure/azure-webjobs-sdk/issues/995](https://github.com/Azure/azure-webjobs-sdk/issues/995) 
We can remove ICloudBlobStreamBinder  totally separately - that was just more churn and breaks and this change was already pretty large.